### ISSUE TITLE
Local timeline RSS feed

### ISSUE THREAD

**Issue by mossymossmoss** [mastodon/mastodon#17269]
### Pitch

Add the ability to get an RSS feed of all public posts on a specific instance

### Motivation

Useful to have new posts linked to on a Discord/Matrix channel with the use of a bot

**Comment by mhkhung** [comment]
For now, use the json..
```
.../api/v1/timelines/public?local=true&only_media=false
```

**Comment by markcellus** [comment]
Yeah, using the [Mastodon API](https://docs.joinmastodon.org/client/intro/) directly to display timeline content is probably the best way to go if you want to display in a chat app. RSS feeds were meant to be consumed by RSS Readers.  But adding more RSS support in https://github.com/mastodon/mastodon/issues/21500 would be great!

**Comment by sanderhollaar** [comment]
Example:
```
curl -s https://social.overheid.nl/api/v1/timelines/public?local=true | \
jq "reverse | .[] | select(.created_at|contains(\"$(date +%Y-%m-%dT)\")) | [.created_at,.content]" | \
elinks -dump -force-html -no-numbering -no-references
```
